code-challenges.test.js

```diff
@@ -22,8 +22,25 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // b) Create the function that makes the test pass.
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
-
-// a) Create a test with expect statements for each of the variables provided.
+//pseudocode:
+//input:1 object
+//output: return string with interpolation using the original input.
+//create a function divByThree using dot notation we will call the number in the object to check if divisible by three. Using the modulo and an equality operator to check if the number is divisible by three if there is no remainder the number turns out to be divisible by three. The modulo operator displays remainders and the equality operator checks that it is strictly equal to zero (no remainder). Then it returns an interpolated string to let the user know if the input is divisible by three or not.
+
+// // a) Create a test with expect statements for each of the variables provided.
+describe("divByThree", () => {
+  it("should return '15 is divisible by three' when given { number: 15 }", () => {
+    expect(divByThree({ number: 15 })).toBe("15 is divisible by three")
```

Comment:
You can have one it block for these three test cases, and check the differences in the expect. Why did you use toBe as the matcher?